packages/office-ui-fabric-react/src/components/Icon/examples/Icon.SvgFactory.Example.tsx

```diff
@@ -0,0 +1,30 @@
+// tslint:disable:jsx-wrap-multiline max-line-length
+import * as React from 'react';
+import { OneDriveIcon, YammerIcon, BorderBlindsIcon } from '@fluentui/react-icons';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const iconClass = mergeStyles({
```

Comment:
This is fine temporarily, but I'd like to move to raw styling. We could stage this change, so this is a fine short term implementation.

Reply:
Calling mergeStyles here is unnecessary if you're also calling it in the render function. I would suggest removing it from the render function if possible.